browser/main/TopBar/index.js

```diff
@@ -244,6 +250,14 @@ class TopBar extends React.Component {
     })
   }
 
+  handleFocusSearch () {
```

Comment:
Oh, I missed it.
How about renaming existing `handleSearchFocus` to `handleOnSearchFocus`?